### PR TITLE
Redacting or eliding text in case

### PR DESCRIPTION
**Pull request by anastasia**
Screen Shot 2019-12-11 at 1 45 46 PM

### CODE REVIEW

File: capstone/capdb/models.py [hunk 1]
```diff
@@ -915,6 +921,14 @@ class Meta:
     def save(self, *args, **kwargs):
         if self.in_scope != self.get_in_scope():
             self.in_scope = not self.in_scope
+
+        # Add a custom footer message to instance if redactions or elisions exist but no text is provided
+        if not self.custom_footer_message and (self.no_index_redacted or self.no_index_elided):
```
Comment (jcushman) [inline capstone/capdb/models.py:926]: I would do this in the view so it's easier to change later.

**Review by jcushman — COMMENTED** [review]
Nice work! One suggested change ...

File: capstone/capdb/models.py (hunk 1 above)
Comment (anastasia) [reply capstone/capdb/models.py:926]: If we do this in the view, we're going to have to do another case lookup. I'm trying to keep case lookups to a minimum.... Do you think that's needless to worry about?

**Comment by jcushman** [comment]
Let's try this out on beta while we continue to chat about that one nit ...

**Pull request merged**